src/Presentation/SkinManager.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\Presentation;
+
+use InvalidArgumentException;
+
+class SkinManager {
```

Comment:
I'm not sure if this class "manages" anything, it looks like a fancy mutable value object to me. Maybe call it `SkinSettings`, `SkinConfiguration` or just `Skins`?